.gitignore

```diff
@@ -17,3 +17,6 @@ MANIFEST
 barcode/version.py
 tests/index.html
 tests/test_outputs/
+.vscode/*
```

Comment:
I'd suggest you add this entry to `~/.config/git/ignore`, so that git will ignore it for _all_ repositories that you work on locally, and you don't need to go around adding it everywhere.

```suggestion
```